src/components/twilio-voice-dialer/routes.js

```diff
@@ -0,0 +1,64 @@
+import { Router } from 'express';
+import crypto from 'crypto';
+import Twilio from 'twilio';
+import config from '../../config.js';
+
+const AccessToken = Twilio.jwt.AccessToken;
+const VoiceGrant = AccessToken.VoiceGrant;
+const VoiceResponse = Twilio.twiml.VoiceResponse;
+
+const router = Router();
+const { appSid, accountSid, apiKeySid, apiKeySecret, callerId, callbackBaseURL } = config;
+const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
+
+router.get('/token', (req, res) => {
+  const identity = req.query.identity;
+  const token = new AccessToken(accountSid, apiKeySid, apiKeySecret, { identity, ttl: 3600 });
+  const voiceGrant = new VoiceGrant({
+    outgoingApplicationSid: appSid,
+    incomingAllow: !!identity,
+  });
+
+  token.addGrant(voiceGrant);
+  res.send({ token: token.toJwt() });
+});
+
+router.post('/twiml', (req, res) => {
+  const twiml = new VoiceResponse();
+  const dial = twiml.dial();
+  
+  // Generates 1:1 conference
+  const roomName = `conference-${crypto.randomUUID()}`;
+
+  let recipient = req.body.recipient;
+  recipient = /^[\d\+\-\(\) ]+$/.test(recipient) ? recipient : 'client:' + recipient;
```

Comment:
This checks if the recipient is a PSTN number right? Can we put a comment here making that clear, just for posterity.

Reply:
I'm also hesitant to change the function behavior based on regex, I used a custom parameter with value `'client' | 'pstn'` in the Voice JS Softphone, and then just changed behavior based on that. What do you think?

Reply:
I think it's simpler if we auto detect a phone number vs client for the customer. If we change it to a parameter, are you also suggesting we change our web component to allow users to click pstn or client?

Reply:
Good point, it makes the UI unnecessarily complicated. Let's just use a regex.